packages/wallet-connector/src/connectors/ethereum/iframe.ts

```diff
@@ -0,0 +1,99 @@
+import { combineLatest, defer, Observable } from "rxjs"
+import { distinctUntilChanged, first, map, mergeMap, shareReplay, startWith } from "rxjs/operators"
+import Web3 from "web3"
+import { AbstractConnectionProvider } from "../../provider"
+import type { Maybe } from "../../common/utils"
+import { cache, promiseToObservable } from "../../common/utils"
+import type { ConnectionState } from "../../connection-state"
+import { getStateConnected, getStateConnecting, getStateDisconnected } from "../../connection-state"
+import { Blockchain } from "../../common/provider-wallet"
+import type { EthereumProviderConnectionResult } from "./domain"
+
+type IframeInstance = any
```

Comment:
Is it possible to replace "any" types for IframeInstance and MewInstance?

Reply:
mew dont provide any types